### ISSUE TITLE
[Feature]: Add in helper methods for rotation/translation message building

### ISSUE THREAD

**Issue by LRTNZ** [Simpit-team/KerbalSimpitRevamped-Arduino#9]
## Is your feature request related to a problem? Please describe.

Currently when a user of this library wants to create a rotation/translation message to send KSP, they have to set  
the mask values in the message themselves, wherever/whenever they create one of those messages.  
This is not very clean, and is less user friendly than it should be.

## Describe the solution you'd like

To implement methods where you can pass in the desired rotation/translation axis values, and the result will
be a rotation/translation message, with the required bits set.

**Comment by CallumDixon** [comment]
Do you have a solution in  mind? tbh i quite like the current implementation, ie you can tell simpit the values you want to change etc, and it wont interfere with other ones. But if you have a better idea in mind then id sure like to hear it

**Comment by LRTNZ** [comment]
Well currently whenever you set a rotation axis, you also have to set the relevant flag in your code. What would be better would be something where you can do:  
```
MessageType message;
message = setPitch(message, pitch value);
message = setRoll(message, roll value);
mySimpit.send(MessageType, message);
```

Which I feel is much cleaner than:  
```
MessageType message;
message.pitch = pitch value;
message.roll = roll value;
message.mask = PITCH_MASK + ROLL MASK;
mySimpit.send(MessageType, message);
```
That way behind the scenes we can change the way the messaging is implemented, without causing issues for the libraries end users.
It also results in cleaner code, which helps a great deal in understanding the code and what is meant to be going on - especially when it comes to helping with debugging issues, as it is easier to see what is happening in the code as an outsider.

**Comment by PBechon** [comment]
Closed as #12  was merged
